client/my-sites/themes/themes-banner/style.scss

```diff
@@ -79,8 +81,10 @@
 		right: 0;
 		top: 0;
 		width: 1.7em;
+		z-index: 1;
 
 		&:hover {
+			background: rgba( white, 0.8 );
```

Comment:
What's going on here?

Reply:
I don't know but they are _all over_ that file. I've removed all references to `white` and `black` and converted them to `rgba()` or hexcode where relevant.